extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -104,6 +110,20 @@ class AmpFitText extends AMP.BaseElement {
         return this.textContent_ || this.contentWrapper_.textContent;
       },
     });
+
+    if (window.ResizeObserver) {
```

Comment:
no need to register in `buildCallback` if you already have it in `layoutCallback`